<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="G10GNLT041" name="com.arcapi.Buildingtestcases.OtherFuelMeterCreateAPITest" tests="1" failures="0" timestamp="20 Dec 2018 09:55:00 GMT" time="452.917" errors="1">
  <testcase name="OtherFuelMeterCreateAPI" time="452.917" classname="com.arcapi.Buildingtestcases.OtherFuelMeterCreateAPITest">
    <error type="com.jayway.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[com.jayway.restassured.path.json.exception.JsonPathException: Failed to parse the JSON documentat com.jayway.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)at com.jayway.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:893)at com.jayway.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:972)at com.jayway.restassured.path.json.JsonPath.get(JsonPath.java:201)at com.jayway.restassured.path.json.JsonPath$get$0.call(Unknown Source)at com.jayway.restassured.internal.RestAssuredResponseOptionsGroovyImpl.path(RestAssuredResponseOptionsGroovyImpl.groovy:392)at com.jayway.restassured.internal.RestAssuredResponseOptionsImpl.path(RestAssuredResponseOptionsImpl.java:195)at com.arcapi.Buildingtestcases.OtherFuelMeterCreateAPITest.OtherFuelMeterCreateAPI(OtherFuelMeterCreateAPITest.java:80)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1183)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.at groovy.json.JsonLexer.nextToken(JsonLexer.java:85)at groovy.json.JsonLexer$nextToken.call(Unknown Source)at com.jayway.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:96)at com.jayway.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)at com.jayway.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:82)at com.jayway.restassured.path.json.JsonPath$4$1.method(JsonPath.java:891)at com.jayway.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:926)... 33 more]]>
    </error>
  </testcase> <!-- OtherFuelMeterCreateAPI -->
</testsuite> <!-- com.arcapi.Buildingtestcases.OtherFuelMeterCreateAPITest -->
